### PR TITLE
Add income and employment data to 1171 PDF

### PR DESCRIPTION
**Pull request by jessieay**
* For up to 2 employed and 2 self-employed household members

### CODE REVIEW

**Comment by jessieay** [comment]
[Fill in DC 1171 with Phase II data](https://trello.com/c/W9Cjeuyb/116-fill-in-dc-1171-with-data)

File: app/models/snap_application.rb [hunk 1]
```diff
@@ -2,6 +2,20 @@ class SnapApplication < ApplicationRecord
...
+      income_child_support,
+      income_foster_care,
+      income_other,
+      income_pension,
+      income_social_security,
+      income_ssi_or_disability,
+      income_unemployment_insurance,
+      income_workers_compensation,
+      members.map(&:monthly_income),
+    ].flatten.compact.reduce(:+)
```
Comment (jayroh) [inline app/models/snap_application.rb]: I love using stuff like `reduce(:+)`. Always feels like ✨ magic ✨

File: spec/models/employed_member_attributes_spec.rb [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+      position = "second"
+
+      result = EmployedMemberAttributes.new(
+        member: member,
+        position: position,
+      ).to_h
+
+      expect(result).to eq(
+        second_employed_full_name: "Foo Flace",
+        second_employed_employer_name: "McTacoBacon R Us",
```
Comment (jayroh) [inline spec/models/employed_member_attributes_spec.rb]: 😆

File: spec/models/member_spec.rb [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+      it "returns monthly income" do
+        member = build(
+          :member,
+          employment_status: "employed",
+          employed_pay_quantity: 10,
+          employed_pay_interval: "Hour",
+          employed_hours_per_week: 10,
+        )
+
+        expect(member.monthly_income).to eq 433.0
```
Comment (jayroh) [inline spec/models/member_spec.rb]: Would it be worth changing the `433.0` to a variable that has the math that builds this number assigned to it?  Right now `433.0` is a bit magic-number'y

File: spec/models/snap_application_attributes_spec.rb [hunk 4]
```diff
@@ -36,6 +37,19 @@
         residential_address_zip: "12345",
         signature: "Mr. RJD2",
         signature_date: snap_application.signed_at.to_s,
+        total_money: snap_application.total_money,
+        monthly_gross_income: snap_application.monthly_gross_income,
+        financial_accounts_checking_or_savings_account: false,
+        financial_accounts_life_insurance: false,
+        financial_accounts_other: false,
+        financial_accounts_mutual_funds_or_stocks: false,
+        financial_accounts_for_oh_one_k_or_iras: true,
```
Comment (jayroh) [inline spec/models/snap_application_attributes_spec.rb]: `for_oh_one_k` or `four_oh_one_k`?

Can we not use `401k` instead?

Comment (jessieay) [inline spec/models/snap_application_attributes_spec.rb]: eagle 👁  -- thanks for catching this!

File: spec/models/member_spec.rb (hunk 3 above)
Comment (jessieay) [inline spec/models/member_spec.rb]: love it -- updated at 5d5c3cf

**Pull request merged**